### DISCUSSION TITLE
Don't disable scroll when Select is open (for Style Guide)

### DISCUSSION THREAD

**Discussion by maybebansky** [radix-ui/themes#481] [General]
I've set `defaultOpen={true}` on a Select element in my Style Guide page so I can demo how it looks when open. 

This works however the page scroll is disabled which I don't want in the Style Guide. Its doubly confusing as I have multiple Selects on this page which are open, so you need to close all of them to scroll. 

Is there a way of disabling this behaviour? Or is there another way of demoing how the open state looks?

**Comment by kevinmitch14** [comment]
Looks like this is just how the Select Primitive works right now. https://github.com/radix-ui/primitives/issues/1496. 

If it's just to showcase the component then maybe some of the workarounds in that issue could serve you.
